src/utils/styleUtils.ts

```diff
@@ -26,3 +26,16 @@ export function calcAvatarMeetingColor(fromColor: string): string {
 	};
 	return toColorString(highlightRegular);
 }
+
+export const calcScaleDivisor = (): number => {
+	const localStorage = window.parent.localStorage.getItem('settings');
+	if (localStorage) {
+		const settingsStorage = JSON.parse(window.parent.localStorage.getItem('settings') || '')[
```

Comment:
You can avoid to repeat `window.parent.localStorage.getItem('settings')`

`const settingsStorage = JSON.parse(localStorage)['settings.appearance_setting.scaling'];`